addons/web/static/src/xml/base.xml

```diff
@@ -239,7 +239,9 @@
     <div class="dropdown-divider o_debug_leave_section" role="separator"/>
     <a role="menuitem" href="#" data-action="split_assets" class="dropdown-item">Activate Assets Debugging</a>
     <a role="menuitem" href="#" data-action="regenerateAssets" class="dropdown-item">Regenerate Assets Bundles</a>
-    <a t-if="manager._is_admin" role="menuitem" href="/web/become" class="dropdown-item">Become Superuser</a>
+    <a t-if="manager._is_admin &amp;&amp; !manager._is_superuser" role="menuitem" href="/web/become" class="dropdown-item">Become Superuser</a>
```

Comment:
This change has been done to Hide `Become Superuser` when user is already _Superuser_.
![Screenshot from 2019-05-02 00-21-33](https://user-images.githubusercontent.com/16624719/57035696-4cac2d80-6c70-11e9-95bf-ba767a9577b8.png)